crowd-server/crowd_server/apps/food_fact/serializers.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+
+
+class UserInSerialiser(serializers.Serializer):
+    
+    question_id = serializers.CharField (required=True,write_only=True)         #if write_only = True : Never show in response 
```

Comment:
After rewriting the model, rewrite accordingly. Her ModelSerializer is better choice